modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -264,6 +268,19 @@ private void _assertObjectRelationshipEndpoints(
 		}
 	}
 
+	private void _assertOpenAPIYAML(
```

Comment:
It would be great if this test could assert more, not only that a schema is contained. Ideally it should check the whole yaml. I know that is hard. Maybe we can transform it to json and compare it to the JSON version?

Reply:
hey @4lejandrito! @ccorreagg and I were trying to make this possible but it doesn't work due to there is a class that is not prepared using the gradle dependencies for testing. As well as, we also try it including it in the .bnd file and it is not also possible. Did we leave this solution or is there any suggestion to test this?

Reply:
never mind! @ccorreagg is amazing :heart: